### DISCUSSION TITLE
Deploying an express server with file upload returning server error with Now

### DISCUSSION THREAD

**Discussion by Segun98** [vercel/vercel#4470] [Help] [answered]
How do I successfully deploy my express server with file upload using Multer?

index.js
![Screenshot (72)](https://user-images.githubusercontent.com/53624548/82740834-4473d400-9d44-11ea-9227-f8f8c7fd4cfe.png)

now.json file
![Screenshot (73)](https://user-images.githubusercontent.com/53624548/82740847-5e151b80-9d44-11ea-9a74-22d9f3797822.png)

**Comment by mcsdevv** [comment, marked as answer]
Hello @Segun98, Vercel is a serverless platform designed for static frontends and Serverless Functions. As a result, it is not possible to run an Express server.

You can find more information about this here: https://vercel.com/knowledge/npm-run-start-not-working

You may also want to look at Serverless Functions: https://vercel.com/docs/v2/serverless-functions/introduction

**Reply by mattfranciswork0** [reply]
Isn't it achievable by adding a now.json file in your express server project?

For example, this article covers it:
https://scotch.io/tutorials/easily-deploy-a-serverless-node-app-with-zeit-now
